pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -777,6 +778,14 @@ self: super: {
 
   inherit parinfer-rust;
 
+  phpactor = buildVimPluginFrom2Nix {
+    inherit (phpactor) pname src meta version;
+    postPatch = ''
+      substituteInPlace plugin/phpactor.vim \
+        --replace "g:phpactorpath = expand('<sfile>:p:h') . '/..'" "g:phpactorpath = '${phpactor}'"
```

Comment:
I have no experience with Vim plugin, but I see that you're replacing a path with the path to phpactor.
Therefore, shouldn't that path be `${phpactor}/bin/phpactor` ?

Reply:
`phpactorpath` is simply the path to the phpactor directory

https://github.com/phpactor/phpactor/blob/efe81d28b45b8ab04422711091ca1a3bd681983f/plugin/phpactor.vim#L9

as long as `phpactor#Update()` isn't run this should be fine

https://github.com/phpactor/phpactor/blob/efe81d28b45b8ab04422711091ca1a3bd681983f/autoload/phpactor.vim#L17

Reply:
Oh ok, fair enough, thanks !